LDAR_Sim/src/ldar_sim.py

```diff
@@ -378,6 +384,23 @@ def finalize(self):
                 self.timeseries['datetime'] = Real_datetime
 
             time_df = pd.DataFrame(self.timeseries)
+			# -----------------------------------------------------------------------------
+            #extract the footprints of crews from timeseries
+            fp = time_df['Crew_footprint']
+            
+            schedule = {'route_planning':False,
+                        'init_loc':None,
+                        'homebase':None}
+            
+            for m_label, m_obj in params['methods'].items():
+                if 'scheduling' in m_obj:
+                    if m_obj['scheduling']['route_planning']:
+                        schedule['route_planning'] = True
+                        schedule['init_loc'] = m_obj['scheduling']['LDAR_crew_init_location']
+                        Homebase_file = params['input_directory']/m_obj['scheduling']['home_bases_files']
+                        schedule['homebase'] = pd.read_csv(Homebase_file, sep=',')
+            
+            trajectory_df = build_crew_trajectories(fp, schedule)
```

Comment:
Will this work if there is no routeplanning or mobile scheduling?

Reply:
Also watch capital on Homebase_file

Reply:
Homebase_file chnaged.